clientv3/balancer.go

```diff
@@ -140,27 +143,23 @@ func (b *simpleBalancer) Up(addr grpc.Address) func(error) {
 		return func(err error) {}
 	}
 
-	if len(b.upEps) == 0 {
+	if b.pinAddr == "" {
 		// notify waiting Get()s and pin first connected address
-		close(b.upc)
 		b.pinAddr = addr.Addr
+		close(b.upc)
```

Comment:
should we set pinAddr before we close the channel that triggers Get? I thought the reason we close `b.upc` is because we have a `pinAddr` ready. I think either code will work. However, set address before notify seems more correct to me.